packages/wasm-bundler/wasm/index.js

```diff
@@ -0,0 +1,2 @@
+import { default as wasm0 } from './wasm0.js';
```

Comment:
This is an autogenerated file and it should be included in the `build` or `dist` directory on build time. Should probably add it to `.gitignore.` and/or change the output path in the `index.js` script.

Reply:
Just to clarify, I mean it should end up in the `packages/rust-rng/pkg` directory.

Reply:
sure

Reply:
I want to remove /wasm-bundler/ and pack everything in rust-rng, but first check if it works in snap